app/resolve/cli/__init__.py

```diff
@@ -30,7 +31,12 @@ def save_to_output(result: RquestResult, destination: str) -> None:
 
 def main() -> None:
     args = parser.parse_args()
-    result = execute_query(parser)
+
+    with open(args.body) as body:
+        query_dict = json.load(body)
+    result_modifier = get_results_modifiers_from_str(args.results_modifiers)
```

Comment:
probably my fault in the issue, but this should be plural since it's a list?
```suggestion
    results_modifiers = get_results_modifiers_from_str(args.results_modifiers)
```